pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -775,11 +773,11 @@ func (b *volumeBinder) GetPodVolumes(pod *v1.Pod) (boundClaims []*v1.PersistentV
 	return boundClaims, unboundClaimsDelayBinding, unboundClaimsImmediate, nil
 }
 
-func (b *volumeBinder) checkBoundClaims(claims []*v1.PersistentVolumeClaim, node *v1.Node, pod *v1.Pod) (bool, bool, error) {
+func (b *volumeBinder) checkBoundClaims(logger klog.Logger, claims []*v1.PersistentVolumeClaim, node *v1.Node) (bool, bool, error) {
 	csiNode, err := b.csiNodeLister.Get(node.Name)
 	if err != nil {
 		// TODO: return the error once CSINode is created by default
-		klog.V(4).InfoS("Could not get a CSINode object for the node", "node", klog.KObj(node), "err", err)
+		logger.V(4).Info("Could not get a CSINode object for the node", "err", err)
```

Comment:
See my [other comment](https://github.com/kubernetes/kubernetes/pull/111155#discussion_r935171071): right now, the node must be kept here for now.

This probably comes from a very early revision of my prototype PR where contextual logging was always enabled and the logger was guaranteed to have the node from a `WithValues` call. We don't have that guarantee at the moment.

Reply:
I grabbed the commit https://github.com/kubernetes/kubernetes/pull/110833/commits/01b8046c58e288a0a9d171235fb37d4abe0acc3e to go forward with this PR. 

Since it is not GA and since it can be disabled/enabled, how should we go forward with this? Add the node back and add a TODO to remove it once contextual logging is always enabled by default?

Reply:
Yes, add back all fields. A TODO at each line may be too much, but perhaps create a TODO in the place where you add the WithValues calls and a tracking issue as a reminder to revisit log calls once contextual logging is GA?

Reply:
done :heavy_check_mark: